modules/ROOT/pages/dataweave-cookbook-csv-lookup.adoc

```diff
@@ -40,7 +40,7 @@ CALLING_CODE,COUNTRY_CODE
 .,.
 ----
 
-The following DataWeave script reads each phone number in the payload and uses the `filter` function to search for the calling code within the Mule variable `country_code` that matches the calling code in the phone number. Then the `map` function gets the country code that corresponds to the calling code and returns the results to an array of JSON objects.
+The following DataWeave script reads each phone number in the payload and uses the `filter` function to search for the calling code within the variable `country_code` that matches the calling code in the phone number. Then the `map` function gets the country code that corresponds to the calling code and returns the results to an array of JSON objects.
```

Comment:
this is indeed a Mule variable because it is **vars.country_code**, you could replace this with a DW variable and declare it in the header of the script.